pkg/runtime/application_config_provider.go

```diff
@@ -37,7 +37,9 @@ var flyteAdminConfig = config.MustRegisterSection(flyteAdmin, &interfaces.Applic
 	MetadataStoragePrefix: []string{"metadata", "admin"},
 	EventVersion:          1,
 	AsyncEventsBufferSize: 100,
+	MaxParallelism:        25,
```

Comment:
Have kept this as default for now .
cc : @katrogan @kumare3

Reply:
oh thanks! i missed this earlier